docs/source/deployorderer/ordererdeploy.md

```diff
@@ -0,0 +1,299 @@
+  - **Orderer local MSP (enrollment certificate and private key of the orderer)**
+
+You will either need to use the Fabric CA client to generate the certificates directly into the recommended folder structure or you will need to copy the generated certificates to their recommended folders after they are generated. Whichever method you choose, most users are ultimately likely to script this process so it can be repeated as needed. A list of the certificates and their locations is provided here for your convenience.
+
+If you are using a containerized solution for running your network (which for obvious reasons is a popular choice), **it is a best practice to mount volumes for the certificate directories external to the container where the node itself is running. This will allow the certificates to be used by an ordering node container, regardless whether the ordering node container goes down, becomes corrupted, or is restarted.**
+
+#### TLS certificates
+
+For the ordering node to launch successfully, the locations of the TLS certificates you specified in the [Checklist for a production ordering node](./ordererchecklist.html) must point to the correct certificates. To do this:
+
+- Copy the **TLS CA Root certificate** that contains the public key that is associated with the signing (private) key certificate, which by default is called `ca-cert.pem`, to the orderer organization MSP definition  `organizations/ordererOrganizations/ordererOrg1.example.com/msp/tlscacerts/tls-cert.pem`.
+- Copy the **CA Root certificate** that contains the public key that is associated with the signing (private) key certificate, which by default is called `ca-cert.pem`, to the orderer organization MSP definition  `organizations/ordererOrganizations/ordererOrg1.example.com/msp/cacerts/ca-cert.pem`.
+- When you enroll the orderer identity with the TLS CA, the public key is generated in the `signcerts` folder, and the private key is located in the `keystore` directory. Rename the private key in the `keystore` folder to `orderer0-tls-key.pem` so that it can be easily recognized later as the TLS private key for this node.
+- Copy the orderer TLS certificate and private key files to `organizations/ordererOrganizations/ordererOrg1.example.com/orderers/orderer0.ordererOrg1.example.com/tls`. The path and name of the certificate and private key files correspond to the values of the `General.TLS.Certificate` and `General.TLS.PrivateKey` parameters in the `orderer.yaml`.
+- If you require mutual TLS (`General.TLS.ClientAuthRequired` set to true), you need to indicate to the orderer which TLS CA root certificates to use to authorize clients. Copy the organization's TLS CA root certificate `ca-cert.pem` to `organizations/ordererOrganizations/ordererOrg1.example.com/orderers/orderer0.ordererOrg1.example.com/tls/tls-cert.pem` so that the organization's clients will be authorized.  If `General.TLS.ClientAuthRequired` is set to false, you can skip this step.
```

Comment:
The TLS CA root cert from EACH org that is going to connect to the orderer should be copied, that is, application clients and peers that will connect to this orderer.

Reply:
In this tutorial, there's only one org. But I'll add a clarification note here anyway.